### ISSUE TITLE
textbox: placeholder text fails color contrast requirement in Firefox

### ISSUE THREAD

**Issue by melquan** [eBay/skin#1627]
Due to the opacity Firefox applies to placeholder text by default, the `#707070` applied is showing up as `#AEAEAE` and still does not meet color contrast requirements.

https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder#opaque_text
